demo/rv/README.md

```diff
@@ -123,4 +123,27 @@ $ openssl x509 -in cert.pem -pubkey -noout | openssl enc -base64 -d > example_al
 $ cat example_allowlist_publickey.der | openssl dgst -sha256 | awk '/s/{print toupper($2)}'
 ```
 
-***NOTE:*** Input file cert.pem is an X509 certificate.
\ No newline at end of file
+***NOTE:*** Input file cert.pem is an X509 certificate.
+
+# Configuring RV for HTTPS/TLS Communication
+
+By default, the RV uses HTTP for all communications on port 8040. In addition to that, the RV can be configured to handle HTTPS request from the owner & device on port 8443.
+
+- Generate the Keystore/Certificate for the RV
+
+  * Ensure that the web certificate is issued to the resolvable domain of the Rendezvous server. Refer the above section to generate the keystore.
+
+- Copy the generated Keystore/Certificate to `demo/rv/certs` folder.
+
+- Update the following environment varibles in `demo/rv/rv.env` file
+
+    |  Variable            |  Value            |             description       |
+    | ---------------------|-------------------|-------------------------------|
+    | rv_protocol_scheme  | https             | To enable HTTPS communication.|
+    | rv_https_port       | port number       | The given port number will be used for HTTPS
+    communication. |
```

Comment:
`communication` is rendering into a separate row.

Reply:
Done.